.github/workflows/camunda-helm-integration.yml

```diff
@@ -57,14 +71,14 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": ":alarm: *Helm chart Integration Tests* from `main` failed! :alarm:\n"
+                    "text": ":alarm: *Helm chart Integration Tests* on `main` failed!\n"
                   }
                 },
                 {
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Please check the related workflow execution: https://github.com/camunda/camunda/actions/runs/${{ github.run_id }}"
+                    "text": "Please check this <https://github.com/camunda/camunda/actions/runs/${{ github.run_id }}|GHA workflow run>."
```

Comment:
💭 Maybe out of scope here, but do we want to tag the distro medic in this message? How do the correct people get notified by this alert?

Reply:
Good point. For the next week I'll look at the messages to see how frequently it happens and relay them. If it happens often, I'll adjust to automatically tag the Distro medic after giving them a headsup.